src/Sylius/Behat/Page/Admin/Customer/UpdatePage.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Behat\Page\Admin\Customer;
+
+use Sylius\Behat\Page\Admin\Crud\UpdatePage as BaseUpdatePage;
+
+/**
+ * @author Anna Walasek <anna.walasek@lakion.com>
+ */
+class UpdatePage extends BaseUpdatePage implements UpdatePageInterface
+{
+    /**
+     * @var array
+     */
+    protected $elements = [
+        'email' => '#sylius_customer_email',
+        'first name' => '#sylius_customer_firstName',
+        'last name' => '#sylius_customer_lastName',
+    ];
+
+    /**
+     * {@inheritdoc}
+     */
+    public function isEmailHasValue($elementValue)
```

Comment:
In context you don't have access to page with the elements. 
If you think about assertion customer's email with string from scenario then ok.